pkg/controllers/machinesetsync/machineset_sync_controller_test.go

```diff
@@ -333,10 +383,24 @@ var _ = Describe("With a running MachineSetSync controller", func() {
 				// We could also mock the conversion interface.
 			})
 
-			PContext("when the CAPI machine set does not exist", func() {
-				// The current test is a placeholder and should cover the functionality in,
-				// case authoritativeAPI == machinev1beta1.MachineAuthorityClusterAPI && capiMachineSet == nil,
-				// once that is implemented.
+			Context("when the CAPI machine set does not exist", func() {
+				It("should create the CAPI machine set", func() {
+					capiMs := capiv1resourcebuilder.MachineSet().WithName(mapiMachineSet.Name).WithNamespace(capiNamespace.Name).Build()
```

Comment:
more a question than a nit, but wouldn't this make sense in a `BeforeEach` ? for this context?

Reply:
Here we are not actually "creating it", we are building an object so that we can do k.Get() on it and check if the controller/reconciler created it for us :) Or am I missing your point here?

Reply:
ah yep I got that, my point was just we could still do it in a `BeforeEach()` and that makes it somewhat less surprising when reading through the code :)